core/src/main/java/feast/core/service/JobService.java

```diff
@@ -50,13 +51,13 @@
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
-/** Defines a Job Managemenent Service that allows users to manage feast ingestion jobs. */
+/** A Job Management Service that allows users to manage Feast ingestion jobs. */
 @Slf4j
 @Service
 public class JobService {
-  private JobRepository jobRepository;
-  private SpecService specService;
-  private Map<String, JobManager> jobManagers;
+  private final JobRepository jobRepository;
+  private final SpecService specService;
+  private final Map<Runner, JobManager> jobManagers;
```

Comment:
Out of scope for this PR, but I still feel like JobManager can be changed to something closer to Runner. In fact it can probably be `Runner` itself. Meaning this code would become `private final Map<RunnerType, Runner>`